forch/faucetizer.py

```diff
@@ -21,20 +21,20 @@
 
 class Faucetizer:
     """Collect Faucet information and generate ACLs"""
+    # pylint: disable=too-many-arguments
     def __init__(self, orch_config, structural_config_file, segments_to_vlans,
-                 behavioral_config_file):
+                 behavioral_config_file, reschedule_acl_file_handlers=None):
         self._dynamic_devices = {}
         self._static_devices = {}
         self._segments_to_vlans = segments_to_vlans
         self._structural_faucet_config = None
         self._behavioral_faucet_config = None
-        self._acl_configs = {}
-        self._next_cookie = 1
+        self._next_cookie = None
         self._config = orch_config
         self._structural_config_file = structural_config_file
         self._behavioral_config_file = behavioral_config_file
+        self._reschedule_acl_file_handlers = reschedule_acl_file_handlers
```

Comment:
the use of the word "reschedule" is very strange here.  What are you trying to say?  Make sure you use a descriptive word of what it is.  I think part of the problem is you're perpetuating the use of the term "schedule" from the observer class... which makes no sense (there is no schedule involved).  So, I wouldn't perpetuate their bad terminology outside of the file watcher class.

Reply:
This function is supposed to unregister existing handlers for all ACL files and register new handlers for new ACL files. Maybe refresh_acl_file_handlers?

Reply:
reregister?

Reply:
reregister_ or refresh_ would be fine

Reply:
Using _reregister_acl_file_handlers now.